tensorflow/core/kernels/matmul_op_impl.h

```diff
@@ -990,10 +1000,9 @@ class BaseBatchMatMulOp : public OpKernel {
 
       LaunchBatchMatMul<Device, float>::Launch(
           ctx, in0_reshaped_float, in1_reshaped_float, adj_x_, adj_y_, trans_x_,
-          trans_y_, bcast, &out_reshaped_float);
-      FastConvertFromFloat<Tout>(out_reshaped_float.flat<float>().data(),
-                                 out_reshaped.flat<Tout>().data(),
-                                 out->NumElements());
+          trans_y_,grad_input_1_, grad_input_2_,  bcast, &out_reshaped_float);
```

Comment:
nit: there's a formatting issue here.  Can you run the formatter?

Reply:
Done!